pytypes.toml

```diff
@@ -3,6 +3,6 @@ unifier = []
 [pytypes]
 project = "repo"
 stdlib_path = "/usr/lib/python3.10"
-proj_path = "/home/benji/Documents/Uni/heidelberg/04/pytype/repo"
-venv_path = "/home/benji/.cache/pypoetry/virtualenvs/pytypes-xvtnrWJT-py3.10"
+proj_path = "/home/secret/project-dir"
+venv_path = "/home/secret/.cache/pypoetry/virtualenvs/pytypes-xvtnrWJT-py3.10"
```

Comment:
Drop the properties. Each user has different values for the paths and will understand with the error that proj_path & venv_path have to be defined if these values are missing.

Reply:
This is meant to be an example config file. It is standard in any large project to put such an example in the root of the project

Reply:
Sample unifiers have been added as requested.